opencensus/common/monitored_resource/gcp_metadata_config.py

```diff
@@ -155,4 +159,8 @@ def _get_attribute(attribute_key):
             # to a normal string:
             attribute_value = attribute_value.decode('utf-8')
 
+        transformation = _ATTRIBUTE_TRANSFORMATIONS.get(attribute_key)
```

Comment:
instead of this transform, this could be structured like the java version, creating a method for each of the values and then add any transformation the the method for the given value.
https://github.com/census-instrumentation/opencensus-java/blob/25475e71ecf31faa3fbf57ca8077a67d9246318f/contrib/resource_util/src/main/java/io/opencensus/contrib/resource/util/GcpMetadataConfig.java